src/components/Header.tsx

```diff
@@ -2,28 +2,44 @@ import React from 'react';
 import AppBar from '@material-ui/core/AppBar';
 import Toolbar from '@material-ui/core/Toolbar';
 import IconButton from '@material-ui/core/IconButton';
+import Grid from '@material-ui/core/Grid';
 import MenuIcon from '@material-ui/icons/Menu';
 import { Link } from 'react-router-dom';
+import styled from 'styled-components';
 
-const onMenuClick = (e: React.MouseEvent<HTMLElement, MouseEvent>) => {
-  //handle when the menu button is clicked here
-};
+import Logo from './Logo';
+import { Container } from '../App.styles';
+
+const headerVerticalMargin = 24;
 
-function Header() {
+const StyledHeader = styled(AppBar)`
+  && {
+    background: none;
+    box-shadow: none;
+    margin-top: ${headerVerticalMargin}px;
+    margin-bottom: ${headerVerticalMargin / 2}px;
+  }
+` as typeof AppBar;
```

Comment:
This is probably what I needed to make TS happy.

Reply:
Yep that is exactly why that is there 😅